Base-MLP_1.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import sklearn\n",
+    "import csv\n",
+    "import matplotlib.pyplot as plt\n",
+    "from sklearn.neural_network import MLPClassifier\n",
+    "from sklearn.metrics import confusion_matrix, plot_confusion_matrix"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_1 = pd.read_csv('data/info_1.csv', index_col=0)\n",
+    "train_df = pd.read_csv('data/train_1.csv', header=None)\n",
+    "test_df = pd.read_csv('data/train_1.csv', header=None)\n",
```

Comment:
wrong file loaded

Reply:
Agree, uploaded file should produce a separate output file.